src/util/ssa_expr.h

```diff
@@ -18,7 +18,7 @@ class ssa_exprt:public symbol_exprt
 public:
   /// Constructor
   /// \param expr: Expression to be converted to SSA symbol
-  explicit ssa_exprt(const exprt &expr) : symbol_exprt(irep_idt(), expr.type())
+  explicit ssa_exprt(const exprt &expr) : symbol_exprt(expr.type())
   {
```

Comment:
I kind of liked that `symbol_exprt` now enforces that you set the identifier.

Reply:
I don't think `symbol_exprt` actually does? This change here just removes an unnecessary call to `set_identifier`.

Reply:
No, it doesn't, but I was hoping it would eventually.

Reply:
We can certainly put the `irep_idt()` back if and when we do.